ngx-tools/type-guards/README.md

```diff
@@ -114,6 +132,23 @@ isFunction('foo');    // Returns: false
 ```
 
 
+## `isHTMLInputElement`
+
+[[source]](src/is-html-input-element/is-html-input-element.ts)
+
+Determine if a value is an HTML input element.
```

Comment:
I'm splitting hairs, but when I see 'value' and 'html input' together, I'm assuming the value attribute of the input, so I initially misunderstood this sentence.

Maybe "Determine if an element is an HTML `<input>`"

Reply:
I almost worded it that way, but the issue is that when this is used, by design, it _is_ just value. The function isn't technically accepting an element, rather a JavaScript value. The function then determines that the value is actually a Dom element.

All type-guards accept _any_ value and then return a specific type.